docs/guides/nrfconnect_factory_data_configuration.md

```diff
@@ -108,6 +108,7 @@ The following table lists the parameters of a factory data set:
 | `spake2_verifier` |           SPAKE2+ verifier           |    97 B    | byte string  |  mandatory  |                                                                                                                                                                        The SPAKE2+ verifier generated using SPAKE2+ salt, iteration counter, and passcode.                                                                                                                                                                         |
 |  `discriminator`  |            Discriminator             |    2 B     |    uint16    |  mandatory  |                                                                                                                                                   A 12-bit value matching the field of the same name in the setup code. The discriminator is used during the discovery process.                                                                                                                                                    |
 |    `passcode`     |            SPAKE passcode            |    4 B     |    uint32    |  optional   | A pairing passcode is a 27-bit unsigned integer which serves as a proof of possession during the commissioning. Its value must be restricted to the values from `0x0000001` to `0x5F5E0FE` (`00000001` to `99999998` in decimal), excluding the following invalid passcode values: `00000000`, `11111111`, `22222222`, `33333333`, `44444444`, `55555555`, `66666666`, `77777777`, `88888888`, `99999999`, `12345678`, `87654321`. |
+|   `product_appearance`    | Product visible appearance | 2 B | JSON string | optional | The appearance field is a structure that describes the visible appearance of the product. This field is provided in the JSON format and consist of two attributes: `finish` (1B), `primary_color` (1B). See the [The appearance field description](#the-appearance-field-description) to learn how to set all attributes. | 
```

Comment:
Does this table describe the intermediate JSON format or the factory data on the device? If the latter (which I think is the case based on the previous rows), it should be "CBOR map" rather than "JSON string". Same applies to the row below.